llvm/test/Transforms/SLPVectorizer/RISCV/spillcost.ll

```diff
@@ -45,7 +49,7 @@ baz:
   ret void
 }
 
-; Shouldn't be vectorized
+; Should be vectorized - just one spill of TMP0
```

Comment:
Thanks for fixing the cost over multiple calls. I think this test was supposed to test diamond shaped control flow where the block with the calls wasn't in the tree though. Can we add more tree entires in entry + foo that are used in baz to trigger the cost?